vector/src/test/java/im/vector/app/test/fakes/FakeCreatePollViewStates.kt

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.test.fakes
+
+import im.vector.app.features.poll.PollMode
+import im.vector.app.features.poll.create.CreatePollArgs
+import im.vector.app.features.poll.create.CreatePollViewModel
+import im.vector.app.features.poll.create.CreatePollViewState
+import org.matrix.android.sdk.api.session.events.model.Event
+import org.matrix.android.sdk.api.session.events.model.EventType
+import org.matrix.android.sdk.api.session.events.model.toContent
+import org.matrix.android.sdk.api.session.room.model.message.MessagePollContent
+import org.matrix.android.sdk.api.session.room.model.message.PollAnswer
+import org.matrix.android.sdk.api.session.room.model.message.PollCreationInfo
+import org.matrix.android.sdk.api.session.room.model.message.PollQuestion
+import org.matrix.android.sdk.api.session.room.sender.SenderInfo
+import org.matrix.android.sdk.api.session.room.timeline.TimelineEvent
+import kotlin.random.Random
+
+object FakeCreatePollViewStates {
```

Comment:
minor naming comment for fakes vs fixtures 

Fakes are objects where behaviour has been changed/overridden or they're extended with test helpers for capturing information, whereas a fixture is a reusable set of defaults, 

I would argue the content of the file fits more into the description of a fixture 

for example a `FakeRoom` could return a `FakeTimeline` which returns a list of `PollEvent fixtures`, usually it's case of `class = fake`, `data class = fixture`